src/components/contextual/pages/pool/staking/StakingIncentivesCard.vue

```diff
@@ -14,18 +14,23 @@ import { StakeAction } from '@/components/contextual/pages/pool/staking/StakePre
 import { usePoolStaking } from '@/providers/local/pool-staking.provider';
 
 import { deprecatedDetails } from '@/composables/usePoolHelpers';
+import { usePoolWarning } from '@/composables/usePoolWarning';
 
 type Props = {
   pool: Pool;
 };
 const props = defineProps<Props>();
 
+const emit = defineEmits<{
+  (e: 'setRestakeVisibility', value: boolean): void;
+}>();
 /**
  * STATE
  */
 
 const isStakePreviewVisible = ref(false);
 const stakeAction = ref<StakeAction>('stake');
+const poolId = computed(() => toRef(props, 'pool').value.id);
```

Comment:
Props are reactive so this can just be:
```suggestion
const poolId = computed(() => props.pool.id);
```
But why not just pass `props.pool.id` into `usePoolWarning` below and then there's no need for this computed property?

Reply:
hm, in `usePoolWarning` poolId arg sholud be ref. but not sure that there is really need in it. I will refactor it a bit

Reply:
hm, no, its used in several places, not sure its best idea to refactor it here